js/form/src/DropdownOptions.svelte

```diff
@@ -49,7 +49,7 @@
 			<li
 				class="item"
 				role="button"
-				class:selected={value?.includes(choice)}
+				class:selected={value === choice}
```

Comment:
```suggestion
				class:selected={Array.isArray(value) ? value.includes(choice) : value === choice}
```
So value can either be a single string, or an array of strings in multiselect mode. I believe this is why the original logic used "includes" to handle multiselect mode, but this suggestion should handle both cases.